arviz/tests/base_tests/test_stats.py

```diff
@@ -538,6 +539,13 @@ def test_loo_warning(centered_eight):
         assert loo(centered_eight, pointwise=True) is not None
     assert any("Estimated shape parameter" in str(record.message) for record in records)
 
+@pytest.mark.parametrize("reff", [None, 0.5, 1.0])
+def test_loo_expectation(centered_eight, reff):
+    log_likelihood = get_log_likelihood(centered_eight)
+    log_likelihood = log_likelihood.stack(__sample__=("chain", "draw"))
+    values = np.arange(1, log_likelihood.shape[-1] + 1)
```

Comment:
I think values should the the log likelihood directly, no extra processing or using its shape to create different objects.